posixlib/src/main/resources/scala-native/sys/socket_conversions.c

```diff
@@ -7,7 +7,7 @@
 int scalanative_convert_sockaddr_in(struct scalanative_sockaddr_in *in,
                                     struct sockaddr_in **out, socklen_t *size) {
     struct sockaddr_in *s =
-        (struct sockaddr_in *)malloc(sizeof(struct sockaddr_in));
+        (struct sockaddr_in *)calloc(1, sizeof(struct sockaddr_in));
```

Comment:
Similar to previously expressed concerns: clearing memory which is not a `malloc/immediate_memset` idiom in original code should not happen.